src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/CreateUpdateService.kt

```diff
@@ -59,30 +59,25 @@ class CreateUpdateService(
     val personEntity: PersonEntity = personService.createPersonEntity(person)
     personService.linkRecordToPersonKey(personEntity)
 
-    val processedDataDTO = Person.from(personEntity)
-
     eventLoggingService.recordEventLog(
-      beforePerson = null,
-      processedPerson = processedDataDTO,
-      uuid = personEntity.personKey?.personId?.toString(),
+      beforePersonEntity = null,
+      afterPersonEntity = personEntity,
+      uuid = personEntity.personKey?.personId!!,
       eventType = event,
     )
-
     return personEntity
   }
 
   private fun handlePersonUpdate(person: Person, existingPersonEntity: PersonEntity, event: String?): PersonEntity {
+    val beforePersonEntity = eventLoggingService.snapshotPersonEntity(existingPersonEntity)
```

Comment:
As far as I can see, this method call is compulsory now before calling recordEventLog? Otherwise you run the (considerable) risk of just sending in the entity, which is probably managed and therefore the before and after will be the same.